src/main/java/collab/Main.java

```diff
@@ -1,9 +1,29 @@
 package collab;
 
+import java.util.List;
+
 public class Main {
 
     public static void main(String[] args) {
-        System.out.println("Hello");
-
+        if (args.length !=2) {
+            System.out.println("Usage: Main.exe input.txt output.txt");
```

Comment:
Usage 부분을 java 형태로 표시하는 게 좋지 않을까요?

```suggestion
            System.out.println("Usage: java -jar Collab.jar input.txt output.txt");
```